.rubocop.yml

```diff
@@ -5,7 +5,7 @@ Rails:
   Enabled: true
 
 AllCops:
-  TargetRubyVersion: 2.2
+  TargetRubyVersion: 2.3
```

Comment:
We still support ruby 2.2, if it's giving you an error locally, you have to install rubocop `0.68.1`

Reply:
I know support was dropped for 2.2 but I don't see a reason to make the gem incompatible in the even someone might still be using that ruby version. But do check, if any of our dependencies already dropped support for 2.2 let's remove it. You would also need to change the version in the gemspec file